draft-ietf-quic-transport.md

```diff
@@ -7334,6 +7334,36 @@ change controller of the IETF and a contact of the QUIC working group
 (quic@ietf.org).
 
 
+## QUIC Versions Registry {#iana-version}
+
+IANA \[SHALL add/has added] a registry for "QUIC Versions" under a "QUIC"
+heading.
+
+The "QUIC Versions" registry governs a 32-bit space; see {{versions}}. This
+registry follows the registration policy from {{iana-policy}}. Permanent
+registrations in this registry are assigned using the Specification Required
+policy ({{!RFC8126}}).
```

Comment:
So wasn't this supposed to have two ranges: Wiki page says "The (WIP) QUIC specification reserves 0x00000001 to 0x0000ffff for standardized versions of the protocol." I think this is a minimal Specification Required policy. Then you have the FCFS range 0x0001000 to 0xFFFFFFFF.

Reply:
Based on the way things are for other registries, I figured that it was easier to have the whole space left as specification required.  There is no functional difference other than the involvement of an expert and it simplifies things.  I would rather have someone camp on version 2 than have different rules for parts of an otherwise undifferentiated space.

Reply:
I actually don't see a reason for not separating the space. The are enough codepoints and given we already decided which part of the space we want to use for new versions that are specified in the IETF we should indicate this in the registry, so people know that when they choose a value for testing internal versions.

Reply:
I'm looking for a reason to segregate it.  Protecting IETF rights to little numbers seems like a pretty weak reason when we can use the same rules as everyone else.

Reply:
It's not about protecting, it about letting others know what our plans are. Given there is no shortage on numbers, I only see that as a benefit.

Reply:
Are we agreed on our future plans though? I am interested in using a completely arbitrary number for version 2.

I agree with keeping this flat; we don't need to make our plans known. I would argue instead that, in the interest of greasing, we _shouldn't_ use 0x2 for v2.

Reply:
I'm not sure that 0x1 for v1 was such a great idea either, but we *did* promise.